### PR TITLE
Add Author and Screenshots of Covid19 Live

### PR DESCRIPTION
**Pull request by altevircardosoneto**
*Thanks for submitting a PR to the Snppts ✂️ library!*

## Submission description

Developed in XamarinForms with the aim of tracking Covid-19 cases with data from affected countries, including Brazil.

### CODE REVIEW

File: src/Snppts/Snippets/Covid19Live.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+using System.Collections.Generic;
+
+namespace Snppts.Snippets
+{
+    public class Covid19Live : IAmASnippet
+    {
+        public string Slug => "covid19-live";
+        public string Title => "Covid-19 Live";
+        public string Description => "Developed in XamarinForms with the aim of tracking Covid-19 cases with data from affected countries, including Brazil.";
+        public GitHubRepoInfo GitHubRepoInfo => new GitHubRepoInfo("https://github.com/Altevir/xamarinforms-covid19");
```
Comment (jfversluis) [inline src/Snppts/Snippets/Covid19Live.cs]: No need for the github domain. 
Just `Altevir/xamarinforms-covid19` is enough

File: src/Snppts/Snppts.csproj [hunk 2]
```diff
@@ -24,7 +24,5 @@
     <Folder Include="Views\Shared\Components\PopularCategories\" />
     <Folder Include="Views\Shared\Components\AllCategories\" />
     <Folder Include="Views\Shared\Components\RandomSnippets\" />
-	<Folder Include="Snippets\" />
```
Comment (jfversluis) [inline src/Snppts/Snppts.csproj:27]: Can you undo the changes in the csproj file?

**Review by jfversluis — CHANGES_REQUESTED** [review]

File: src/Snppts/Snippets/Covid19Live.cs (hunk 1 above)
Comment (altevircardosoneto) [reply src/Snppts/Snippets/Covid19Live.cs]: ops, sorry, i will fix.

File: src/Snppts/Snppts.csproj (hunk 2 above)
Comment (altevircardosoneto) [reply src/Snppts/Snppts.csproj:27]: yeap, i will remove.

Comment (altevircardosoneto) [reply src/Snppts/Snppts.csproj:27]: in the fork of the project is it not visible to me, something different to do?

File: src/Snppts/Snippets/Covid19Live.cs (hunk 1 above)
Comment (altevircardosoneto) [reply src/Snppts/Snippets/Covid19Live.cs]: removed and committed =)

File: src/Snppts/Snppts.csproj (hunk 2 above)
Comment (jfversluis) [reply src/Snppts/Snppts.csproj:27]: That’s the problem. These lines are removed while they should still be there :)

Comment (altevircardosoneto) [reply src/Snppts/Snppts.csproj:27]: humm, sorry, is it necessary to send a new PR?

Comment (jfversluis) [reply src/Snppts/Snppts.csproj:27]: It should not! You can either revert this change in a new commit or simply add back these lines in a new commit

Comment (altevircardosoneto) [reply src/Snppts/Snppts.csproj:27]: added again and made a new commit

**Review by jfversluis — APPROVED** [review]
